### PR TITLE
Pass process flags to the thread, taking care of inspect ports

### LINKED ISSUE #16: No ability to set thread debug port -- or this is undocumented
It would be handy to be able to configure the debug listener port used by child threads. Right now child threads attempt to use the same port as the parent thread causing EADDRINUSE exception to be thrown.

### PR DESCRIPTION
**Pull request by scinos**
It passes any flag (`execArgv`) from the main process to all threads. When that flag is `--inspect` or `--inspect-brk`, it changes the value of the port so the processes don't try to use the same address. 

This behaviour can be disabled by the user by passing a custom `execArgv`.

Bonus: fixes the build process in local

Fixes #16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +29,12 @@
   },
   "babel": {
     "presets": [
-      "es2015",
-      "es2015-loose"
+      [
```
Comment (andywer) [inline package.json]: 👍
`es2015-loose` was old legacy stuff

**Comment by andywer** [comment]
Awesome, @scinos! Thanks a lot for the PR :)

Looks great! Let's see if one or two more people can have a look at your solution, but this looks very good to me.

**Comment by andywer** [comment]
I merged the updated `master` branch back in, but somehow broke the build 🙈

**Comment by goraSi** [comment]
What's the ETA of this merge to master. Really need this because of Visual Studio Code.

**Comment by andywer** [comment]
@goraSi Not sure, probably next week. Need to find some time to fix the build, but I have little time this weekend. Or maybe @scinos can have a look.

**Comment by scinos** [comment]
@goraSi @andywer Fixed now.

**Comment by andywer** [comment]
Awesome, @scinos! Of course, the options parameter broke the merge... Thanks for fixing!

**Comment by andywer** [comment]
Published as `v0.10.0` 🚀

**Comment by goraSi** [comment]
I have tested new version in Visual Studio Code and it doesn't work beacuse it uses --debug-brk=3296 argument insted of --inspect-brk.

Is it possible to fix this also?

**Comment by scinos** [comment]
I think that's a problem in VSCode config. You might need to specify `protocol: "inspector"` and `port: 9229` in your `launch.json` config.

**Pull request merged**